src/metaswitch/clearwater/cluster_manager/etcd_synchronizer.py

```diff
@@ -271,11 +272,16 @@ def read_from_etcd(self):
             self._last_cluster_view = cluster_view.copy()
 
         except etcd.EtcdKeyError:
+            _log.debug("Key {} doesn't exist in etcd yet".format(self._key))
```

Comment:
Maybe INFO level, but I don't think it's an error or a warning - it will be normal in some cases.